src/net/sourceforge/kolmafia/textui/RuntimeLibrary.java

```diff
@@ -9208,7 +9208,7 @@ private static String getModifierName(final Value arg) {
       return "[" + id + "]";
```

Comment:
instead of special-casing, you can add `SKILL_TYPE` into this `if` to return the `id` directly.

Reply:
good catch,